composer.json

```diff
@@ -11,7 +11,7 @@
     "require": {
         "php": ">=7.2",
         "illuminate/http": "^7.0 || ^8.0",
-        "illuminate/contracts": "^7.0 || ^8.0"
+        "illuminate/contracts": "^7.0 || ^8.0 || ^9.0"
```

Comment:
Maybe we can also remove ^7.0 ?

Reply:
I was thinking it could be a good idea but is not imperative now.. so I would let this change for the future